<launch>
  <node pkg="valve_tracker_triton" type="valve_trainer_node" name="valve_trainer" output="screen">
    <param name="num_hue_bins" value="16"/>
    <param name="num_sat_bins" value="16"/>
    <param name="num_val_bins" value="1"/>
    <param name="mean_filter_size" value="1"/>
    <param name="opening_element_size" value="2"/>
    <param name="closing_element_size" value="12"/>
    <param name="min_value" value="110"/>
    <remap from="stereo" to="stereo_down"/>
    <remap from="image" to="image_rect_color"/>
  </node>
</launch>
